testcases/kernel/syscalls/fallocate/fallocate01.c

```diff
@@ -130,7 +143,9 @@ void cleanup(void)
 		tst_resm(TWARN | TERRNO, "close(%s) failed", fname_mode1);
 	if (close(fd_mode2) == -1)
 		tst_resm(TWARN | TERRNO, "close(%s) failed", fname_mode2);
-	tst_rmdir();
+	remove(fname_mode1);
+       remove(fname_mode2);
+       rmdir(tempdir);
```

Comment:
indentation problem. before "git add", if you do "git diff" for a changed file, you can ctach the indentation issues.